.pre-commit-config.yaml

```diff
@@ -84,12 +79,12 @@ repos:
     hooks:
       - id: black
 
-  - repo: https://github.com/PyCQA/flake8
-    rev: 6.0.0
+  - repo: https://github.com/charliermarsh/ruff-pre-commit
+    rev: "v0.0.161"
     hooks:
-      - id: flake8
+      - id: ruff
         args:
-          - --count
+          - --fix
```

Comment:
This could be changed to a single line:
```
args: ["--fix"]
```

IMOP this feels cleaner.

Reply:
SGTM.